jupyter-lib/shared-compiler/src/main/kotlin/org/jetbrains/kotlinx/jupyter/compiler/util/serializedCompiledScript.kt

```diff
@@ -20,12 +20,22 @@ data class SerializedCompiledScriptsData(
     }
 }
 
+@Serializable
+data class SerializedVariablesState(
+    val name: String = "",
```

Comment:
Do we really need a name here if we have a name as a map key? Or maybe we don't need map key at all?

Reply:
I used key because it's really convenient to get the needed variable fast with only one string value. Perhaps, we might redux and remove the name

Reply:
Ok, let's leave it as is

Reply:
I think we still need to provide accessibility for names without a keymap if a user decides to use it for their own purposes. What do you think of leaving this extra info?